### PR TITLE
Document setting up MLA with existing minio or s3 services

### PR DESCRIPTION
**Pull request by jiachengxu**
fixes: https://github.com/kubermatic/mla/issues/73

### CODE REVIEW

File: content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md [hunk 1]
```diff
@@ -251,6 +273,18 @@ data:
 
 After the new dashboards are applied to the Seed Cluster, they will become available across all Grafana Organizations, and they can be found in the Grafana UI under Dashboards -> Manage.
 
+### Managing Alerting and Recording Rules
+
+Similar to managing Grafana Dashboards, KKP administrators can also define Prometheus-compatible rules for metrics and logs and make them available across all KKP user clusters with MLA enabled.
```
Comment (rastislavs) [inline content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md:278]: I would emphasize the automation a bit: 
 - add "pre-" to "define": "administrators can also pre-define"
 - add "automatically" before "available": "make them automatically available across"

File: content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ Before deploying the MLA stack into the KKP Seed cluster, let’s create two Kub
 helm --namespace mla upgrade --atomic --create-namespace --install mla-secrets charts/mla-secrets --values config/mla-secrets/values.yaml
 ```
 
+This above command will create two Secrets (one for MinIO, and one for Grafana), if want to use your existing Secrets in the Cluster, you can disable the creation by modifying the [mla-secret value.yaml](https://github.com/kubermatic/mla/blob/main/config/mla-secrets/values.yaml#L17-L22)
```
Comment (rastislavs) [inline content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md:43]: - "This above" -> "The above"
- "if want to use " -> "if you want to use"

File: content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md [hunk 3]
```diff
@@ -50,6 +52,8 @@ After the secrets are created, the MLA stack can be deployed by using the helper
 
 This will deploy all MLA stack components with the default settings, which may be sufficient for smaller scale setups (several user clusters). If any customization is needed for any of the components, the steps in the helper script can be manually reproduced with tweaked Helm values. See the “Setup Customization” section for more information.
 
+Also, this will deploy a MinIO instance which will be used by MLA components for storage, if you would like to re-use an existing MinIO instance in your cluster or other S3-compatiable srevices from cloud providers, please refer to [Setting up MLA with Existing MinIO or Other S3-compatiable Services](#setting-up-mla-with-existing-minio-or-other-s3-compatiable-services).
```
Comment (rastislavs) [inline content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md:55]: Start a new sentence with if "or storage. If you would like ..."

**Comment by jiachengxu** [comment]
@rastislavs  I have addressed all the comment above, please let me know if you still want to modify anything

**Pull request merged**